app/request.go

```diff
@@ -2,12 +2,37 @@ package app
 
 import (
 	"context"
+	"fmt"
+	"time"
 
+	"github.com/gosuri/uilive"
+	"github.com/temporalio/tcld/protogen/api/request/v1"
 	"github.com/temporalio/tcld/protogen/api/requestservice/v1"
 	"github.com/urfave/cli/v2"
 	"google.golang.org/grpc"
 )
 
+const (
+	SyncRequestFlagName    = "synchronous-request"
+	RequestTimeoutFlagName = "request-timeout"
+)
+
+var (
+	RequestTimeoutFlag = &cli.DurationFlag{
+		Name:    RequestTimeoutFlagName,
+		Usage:   "Time to wait for asynchronous requests to finish",
```

Comment:
how does this flag work? As in what values are expected?

Is it easier to have this be something like "timeout in seconds" ?

Reply:
It uses the string to duration func in the time package: https://pkg.go.dev/time#ParseDuration
You can pass in values like: 10s, 1h, 1h20m...